### PR TITLE
Doc 410

### PR DESCRIPTION
**Pull request by resmaa**
**Naming conventions for VSphere Region and Zone Tags**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1002#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/00-on-prem-system-requirements.md:84]: We should always include at least a sentence explaining the section

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/00-on-prem-system-requirements.md:56]: This feels like prerequisite

File: content/docs/04-clusters/02-data-center/03-vmware.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/03-vmware.md:81]: This feels like prerequisite

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Minor feedback

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md
Comment (resmaa) [reply content/docs/12-enterprise-version/00-on-prem-system-requirements.md:56]: yes added as an infobox under prerequisites

File: content/docs/04-clusters/02-data-center/03-vmware.md
Comment (resmaa) [reply content/docs/04-clusters/02-data-center/03-vmware.md:81]: yes added as an infobox under the prerequisite section

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1002#pullrequestreview-1198301703" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1002#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
